main/src/cgeo/geocaching/ui/Variable.java

```diff
@@ -0,0 +1,285 @@
+package cgeo.geocaching.ui;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.GridLayout;
+import android.text.Editable;
+import android.text.InputType;
+import android.text.TextWatcher;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.io.Serializable;
+import java.util.List;
+
+import cgeo.geocaching.R;
+
+/**
+ * Created by Slartibartfast on 30/06/2017.
+ *
+ * Class used to display a variable with an equation, such as:
+ *  X = a^2 = b^2
+ */
+
+public class Variable extends LinearLayout {
```

Comment:
What kind of "variable"? Either place this in an own cgeo.geocaching.xxx.calculator package or better rename it (e.g. `CalculatorVariableView`).